### PR TITLE
Hstore fixes and improvements

### PR DESCRIPTION
**Pull request by starstuck**
A few issues we have noticed when running it via production data set

### CODE REVIEW

File: src/query_builder.erl [hunk 1]
```diff
@@ -11,8 +11,8 @@
 
 -include("dqe_idx_pg.hrl").
 
--define(NAMESPACE_PATTERN, "'^(([^:]|\\\\\\\\|\\\\:)+):'").
--define(NAME_PATTERN, "'^(?:[^:]|\\\\\\\\|\\\\:)+:(.*)$'").
+-define(NAMESPACE_PATTERN, "'^(([^:]|\\\\\\\\|\\\\:)*):'").
+-define(NAME_PATTERN, "'^(?:[^:]|\\\\\\\\|\\\\:)*:(.*)$'").
```
Comment (Licenser) [inline src/query_builder.erl:15]: I'm curious why the change from `+` to `*`, is this to allow 'empty' namespaces like `WHERE :color = 'red'` ?

Comment (starstuck) [reply src/query_builder.erl:15]: Exactly for that. Empty namespace is default, so it is actually very likely to be used.

**Comment by Licenser** [comment]
thanks! :D
